python/lvmdrp/functions/run_drp.py

```diff
@@ -286,7 +286,9 @@ def reduce_frame(filename: str, camera: str = None, mjd: int = None,
         kwargs = get_config_options('reduction_steps.detrend_frame', flavor)
         log.info(f'custom configuration parameters for detrend_frame: {repr(kwargs)}')
         detrend_frame(in_image=in_cal, out_image=out_cal,
-                      in_bias=mbias, in_dark=mdark, in_pixelflat=mpixflat, **kwargs)
+                      in_bias=mbias, in_dark=mdark, in_pixelflat=mpixflat,
+                      in_slitmap=Table(fibermap.data) if flavor in {'fiberflat', 'flat', 'object', 'science'} else None,
```

Comment:
I would move this conditional out to a new line to improve readability.  In the line before this one, you could do something like `fmap = Table(fibermap.data) if if flavor in {'fiberflat', 'flat', 'object', 'science'} else None`, then `in_slitmap=fmap` is a bit easier to read.  Or why not move the whole check to inside the `detrend_frame` function?  

Also, is there a reason you need the slitmap as an Astropy Table?  Why not just pass in the fibermap?  The default fibermap is already a fits.BinTableHDU that can be added to a `fits.HDUList`.   The `read_fibermap` function also can return an Astropy Table.

Reply:
Done. Moved the conditional to `detrend_frame`.